### DISCUSSION TITLE
Toggle WiFi on/off with single button

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#847] [Ideas]
Just an idea and not sure if others will want it but I think it would be nice to map a single button to toggle WiFi on/off. For example I only want it when I boot the device and need to set the clock, would like to easily turn it off to save battery after that

**Comment by totriz** [comment]
Or an option to turn on wifi at boot to check for updates and sync the clock and then turn it off automatically until next reboot
